Decisions.MongoDB/DeleteBulkDocumentStep.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using DecisionsFramework;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.Design.Flow.Mapping;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace Decisions.MongoDB
+{
+    [Writable]
+    public class DeleteBulkDocumentStep : BaseDeleteStep
```

Comment:
Let's rename this to `BulkDeleteDocumentStep`

Reply:
I mean from an alphabetical order perspective, it makes sense to name them this way